samples/bluetooth/alexa_gadget/prj.conf

```diff
@@ -13,7 +13,7 @@ CONFIG_ASSERT=y
 CONFIG_LOG=y
 
 
-CONFIG_MAIN_STACK_SIZE=1024
+CONFIG_MAIN_STACK_SIZE=2048
```

Comment:
If this is only for nrf52840 / CC3XX then default main stack size has been updated to handle it, and this would not be needed